.travis.yml

```diff
@@ -7,16 +7,15 @@ cache:
     - $HOME/.composer/cache/files
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
Removing old PHP version is a no-go if we don't change the PHP requirement in the composer.json. Not having CI for these versions means we would not be able to ensure comaptibility, so it is better to tell composer to use an older release for these versions (Behat itself made this exact mistake in 3.4).

Reply:
something like #297 ?